src/main/kotlin/me/partlysanestudios/partlysaneskies/gui/hud/cooldown/TreecapitatorCooldown.kt

```diff
@@ -0,0 +1,60 @@
+//
+// Written by Su386.
+// See LICENSE for copyright and license notices.
+//
+
+
+package me.partlysanestudios.partlysaneskies.gui.hud.cooldown
+
+import me.partlysanestudios.partlysaneskies.PartlySaneSkies
+import me.partlysanestudios.partlysaneskies.data.cache.PetData
+import me.partlysanestudios.partlysaneskies.utils.HypixelUtils
+import me.partlysanestudios.partlysaneskies.utils.MinecraftUtils
+import net.minecraft.init.Items
+import net.minecraft.item.ItemStack
+
+object TreecapitatorCooldown: Cooldown() {
+    override fun getTotalTime(): Long {
+        var cooldown = 2000L
+
+        if (PetData.getCurrentPetName() == "Monkey" && PetData.getCurrentPetRarity().order > PetData.Rarity.LEGENDARY.order && PetData.getCurrentPetLevel() != -1 && (PartlySaneSkies.config?.treecapCooldownMonkeyPet != false)) {
```

Comment:
Pet skins exist, doing an equals check will break with that here

Reply:
It should be fine? I haven't really tested it but I assume because we're using Regex with everything else it should be fine????? Maybe?????? if not i'll look into fixing it

Reply:
Someone has to test it because I dont have pet skins

Reply:
I could test it

Reply:
What's the cheapest petskin

Reply:
Zombee maybe

Reply:
<img width="157" alt="image" src="https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/00c9bb1a-7166-45be-a025-15e68c248005">
it works

Reply:
It actually works now